sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/implementation/jwt/AadJwtEncoder.java

```diff
@@ -141,15 +141,15 @@ private static JWSSigner createSigner(JWK jwk) {
         }
     }
 
-    @SuppressWarnings("unchecked")
+    @SuppressWarnings({"unchecked", "deprecation"})
```

Comment:
If we suppress the warnings here, do we have another issue to track when to remove the deprecations?

Reply:
Opened an issue https://github.com/Azure/azure-sdk-for-java/issues/31571